tsconfig.json

```diff
@@ -41,6 +41,7 @@
     "alwaysStrict": true,
     "strict": true,
     "skipLibCheck": true,
+    "sourceMap": true,
```

Comment:
Remove. Point nyc at the appropriate configuration in the `tsconfig` subfolder. Alternatively or possibly in addition, you should pre-instrument the code by adding a build configuration that outputs the code required for e2e testing so that it runs faster.